examples/data/square01-quad.mesh

```diff
@@ -0,0 +1,61 @@
+MFEM mesh v1.0
```

Comment:
I think it is fine to include this small mesh file, but an alternative is to use the constructor
`static Mesh MakeCartesian2D(int nx, int ny, Element::Type type, bool generate_edges = false, double sx = 1.0, double sy = 1.0, bool sfc_ordering = true);`

Reply:
Thanks, Dylan. I have made the changes as you suggested.

Reply:
Does this mean the new .mesh file can be removed?